cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/view/AzureNetworkView.java

```diff
@@ -5,20 +5,18 @@
 import java.util.Objects;
 import java.util.StringJoiner;
 
-import com.google.common.annotations.VisibleForTesting;
 import com.sequenceiq.cloudbreak.cloud.model.Network;
 import com.sequenceiq.common.model.PrivateEndpointType;
 
 public class AzureNetworkView {
 
-    @VisibleForTesting
-    static final String SUBNETS = "subnets";
+    private static final String SUBNETS = "subnets";
 
-    @VisibleForTesting
-    static final String SUBNET_FOR_PRIVATE_ENDPOINT = "subnetForPrivateEndpoint";
+    private static final String SUBNET_FOR_PRIVATE_ENDPOINT = "subnetForPrivateEndpoint";
 
-    @VisibleForTesting
-    static final String ENDPOINT_TYPE = "endpointType";
+    private static final String ENDPOINT_TYPE = "endpointType";
```

Comment:
Out of curiosity, what is reason behind this change? :smile:

Reply:
Reusing the constants of productive code in tests does not guard against the case when the value of the constant changes. I thus like to introduce constants in tests as separate entities but with the same content. Also, in this way we can make constants private.